README.md

```diff
@@ -188,14 +187,6 @@ Mbed TLS supports being built as a CMake subproject. One can
 use `add_subdirectory()` from a parent CMake project to include Mbed TLS as a
 subproject.
 
-### Microsoft Visual Studio
-
-The build files for Microsoft Visual Studio are generated for Visual Studio 2017.
-
-The solution file `mbedTLS.sln` contains all the basic projects needed to build the library and all the programs. The files in tests are not generated and compiled, as these need Python and perl environments as well. However, the selftest program in `programs/test/` is still available.
```

Comment:
It might be worth noting that these programs are temporarily continuing to use the legacy API. Something like:

"Many of the example programs currently use internal-only functions so may not be very useful as demonstrators. We are currently in the process of migrating them to use only publically-available interfaces."

Reply:
I may be wrong but apart from the program `selftest` that would probably better fit in TF-PSA-Crypto I do not think that the APIs the programs that remain in Mbed TLS are based on have changed a lot. Do you have some specific programs in mind. Your comment is valid in TF-PSA-Crypto but here I am not sure.

Reply:
Actually looking at it I agree - the programs that we still have do not make much use of the legacy API. I withdraw my objection.